File: TT2/dmonica.c
#include <stdio.h>

int main(void){

int id1, id2, idmae;
int soma, id3;
scanf("%d", &idmae);

scanf("%d", &id1);

scanf("%d", &id2);

soma = id1 + id2; 
id3 = idmae - soma ;
if (id1 > id2 && id1 > id3)
{
printf("%d", id1);
}
else if (id2 > id1 && id2 > id3)
{
printf("%d", id2);
}
else
{
printf("%d", id3);
}
return 0;
}

File: TT2/dmonica.java
public class dmonica{
public static void main(String[] args){

int id1;
int id2;
int idmae;
int soma;
int id3;

idmae = MyIO.readInt();
id1 = MyIO.readInt();
id2 = MyIO.readInt();
soma = id1 + id2;

id3 = idmae - soma;

if (id1 > id2 && id1 > id3){
MyIO.println( +id1);
}else if (id2 > id1 && id2 > id3){
MyIO.println( +id2);
}
else if (id3 > id1 && id3 > id2){
MyIO.println( +id3);
}

}
}